app/settings.php

```diff
@@ -57,6 +57,28 @@
                         'pledgerLastName',
                     ],
                 ],
+                [
+                    'templateKey' => 'donor-donation-refund-full',
+                    'subject' => 'Full Donation Refund',
+                    'subjectParams' => [],
+                    'requiredParams' => [
+                        'donorFirstName',
+                        'donationReferenceNumber',
+                        'charityName',
+                        'donationAmount',
+                        'donationTipAmount',
+                    ],
+                ],
+                [
+                    'templateKey' => 'donor-donation-refund-tip',
+                    'subject' => 'Tip Refund',
+                    'subjectParams' => [],
+                    'requiredParams' => [
+                        'donorFirstName',
+                        'donationReferenceNumber',
```

Comment:
I think we should be using `transactionId` here for consistency with the rest of the codebase.